fcrepo-serialization/src/test/resources/binary.rdf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF
+    xmlns:premis="http://www.loc.gov/premis/rdf/v1#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:fedora="http://fedora.info/definitions/v4/repository#">
+
+<rdf:Description rdf:about="http://localhost:8080/rest/binary">
+    <premis:hasFixity rdf:resource="http://localhost:8080/rest/binary#fixity/1434585821367"/>
+    <!-- this is here just for mocking -->
+    <rdf:type rdf:resource="http://fedora.info/definitions/v4/repository#Binary"/>
+</rdf:Description>
+
+<rdf:Description rdf:about="http://localhost:8080/rest/binary#fixity/1434585821367">
+    <rdf:type rdf:resource="http://www.loc.gov/premis/rdf/v1#Fixity"/>
+    <fedora:status rdf:datatype="http://www.w3.org/2001/XMLSchema#string">SUCCESS</fedora:status>
+    <premis:hasMessageDigest rdf:resource="urn:sha1:a8069665cdee87b7963676cfcc759ef0f0c9a0e9"/>
+    <premis:hasSize rdf:datatype="http://www.w3.org/2001/XMLSchema#long">1636261</premis:hasSize>
+    <premis:hasContentLocation rdf:resource="info://org.modeshape.jcr.value.binary.FileSystemBinaryStore@48c53d0b/fcrepo4-data/fcrepo.binary.directory#a8069665cdee87b7963676cfcc759ef0f0c9a0e9"/>
```

Comment:
Actually, this entire response looks like it's from a `/fcr:fixity` request. I would suggest changing it to reflect what is typically returned from a binary resource request.